internal/runners/clean/cache.go

```diff
@@ -76,9 +80,16 @@ func (c *Cache) removeCache(path string, force bool) error {
 		}
 	}
 
-	logging.Debug("Removing cache path: %s", path)
-	err := removeCache(c.path)
+	inUse, err := c.checkPathInUse(path)
 	if err != nil {
+		return errs.Wrap(err, "Failed to check if path is in use")
+	}
+	if inUse {
+		return locale.NewError("err_clean_in_use")
```

Comment:
Should make this an input error (same for the other one below).

Reply:
You updated the other one but not this one 😅

Reply:
🤦‍♂️